reader.py

```diff
@@ -0,0 +1,23 @@
+def main():
+    find_word_frequency()
+
+
+def max_frequency(words):
+    most_frequent = max(words, key=words.get)
+    print(most_frequent, words[most_frequent])
+
+
+def find_word_frequency():

```

Comment:
Nice implementation by using functions!
But - it is better to create functions that doing 1 thing and 1 thing only. this one doing both file opening, fetching words and calculating results

Reply:
I understand the problem with the function doing both the file opening and the calculation, 
but is it necessary to divide the file opening from the fetching as well? Since the fetching is happening while I'm reading the file line by line, I thought it'd be kinda redundant and make the program more complicated than it should be.
That is, unless, you mean I should divide them, then put the fetching function inside of the reading function.

Reply:
it is your call to decide, you should aim for 1 action only, but sometimes it makes code to complicated